torch/csrc/jit/passes/onnx/prepare_division_for_onnx.cpp

```diff
@@ -0,0 +1,36 @@
+#include "torch/csrc/jit/passes/onnx/prepare_division_for_onnx.h"
+#include "torch/csrc/jit/constants.h"
+
+namespace torch { namespace jit {
+
+static void PrepareDivisionForONNXOnBlock(Block* block) {
+  for (auto it = block->nodes().begin(); it != block->nodes().end(); ++it) {
+    for (auto sub : it->blocks()) {
+      PrepareDivisionForONNXOnBlock(sub);
+    }
+    WithInsertPoint guard(*it);
+    auto* subgraph = it->owningGraph();
+
+    if (it->matches("aten::div(int a, int b) -> float")) {
+      // Use onnx::cast before dividing
+      std::vector<Value*> floattensor_inputs = fmap(it->inputs(), [&](Value* input) {
+        Value* longtensor = subgraph->insertNode(subgraph->createNumToTensor(input))->output();
+        // FLOAT = 1
+        // https://github.com/onnx/onnx/blob/6bedd27b0307c9295039bd847895a27275160a98/onnx/onnx.in.proto#L282
+        Node* cast = subgraph->create(onnx::Cast, {longtensor})->i_(attr::to, 1);
```

Comment:
re: zach's comment below, make this emit an `aten::_cast_Float` operator. Then the ONNX pass later can correctly translate this based on which mode it's in (i.e. raw mode will pass that through, but ONNX mode will call the symbolic and insert what you have now)